gradle/libs.versions.toml

```diff
@@ -57,17 +57,20 @@ xdk-oodb = { group = "org.xtclang", name = "lib-oodb", version.ref = "xdk" }
 xdk-web = { group = "org.xtclang", name = "lib-web", version.ref = "xdk" }
 xdk-webauth = { group = "org.xtclang", name = "lib-webauth", version.ref = "xdk" }
 xdk-xenia = { group = "org.xtclang", name = "lib-xenia", version.ref = "xdk" }
+xdk-xunit = { group = "org.xtclang", name = "lib-xunit", version.ref = "xdk" }
 
 javatools = { group = "org.xtclang", name = "javatools", version.ref = "xdk" }
 javatools-unicode = { group = "org.xtclang", name = "javatools-unicode", version.ref = "xdk" }
 javatools-utils = { group = "org.xtclang", name = "javatools-utils", version.ref = "xdk" }
 javatools-turtle = { group = "org.xtclang", name = "javatools-turtle", version.ref = "xdk" }
 javatools-bridge = { group = "org.xtclang", name = "javatools-bridge", version.ref = "xdk" }
 
+hamcrest = { module = "org.hamcrest:hamcrest", version = "2.2" }
 javax-activation = { module = "com.sun.activation:javax.activation", version = "1.2.0" }
 jakarta-xml-bind-api = { module = "jakarta.xml.bind:jakarta.xml.bind-api", version.ref = "jakarta" }
 jaxb-runtime = { module = "org.glassfish.jaxb:jaxb-runtime", version.ref = "jakarta" }
 jline = { module = "org.jline:jline", version.ref = "jline" }
+mockito = {module = "org.mockito:mockito-core", version = "5.11.0"}
 
```

Comment:
Hitting ctrl-alt-L in intelliJ adds trailing spaces for me, so that I don't have to think too much about indentation, but I'm not sure if it's a personal setting or propagated to our codebase. We should work on IDE-environment consitency together with language support. I did add support for the Oracle-Sun Java code standard with checkstyle, but this seems to be a thing no one wants to migrate too. Personally, as long as something is consistent, it doesn't matter how it looks, I can read it. The Coherence standard is the only one I have had trouble doing that with, and it only partly has to do with brackets on single lines makes longer methods and frequent page ups / page downs with a laptop size screen. :-).   We should likely implement a  plugin for an official XTC code standard too, and only have one.